downloads/swan-lake-release-notes/swan-lake-alpha/RELEASE_NOTE.md

```diff
@@ -0,0 +1,1562 @@
+            websub:HubClient hubclient = new (msg);
+            check start notifySubscriber(hubclient);
+        }
+
+        function notifySubscriber(websubhub:HubClient hubclient) returns error? {
+            while (true) {
+                // fetch the message from MB
+                check hubclient->notifyContentDistribution({content: "This is sample content delivery"});
+            }
+        }
+    }
+```
+
+**Publisher Implementation**
+
+The `websub:PublisherClient` is moved to the `websubhub:PublisherClient`.
+
+##### Removed Modules
+
+###### Config
+
+The configuration use cases are now covered under the `configurable` language feature. 
+
+###### Math
+
+The APIs related to random number generation were moved to the new `random` module. The rest of the APIs have replacements in the `lang.float` and `lang.int` packages.
+
+###### Stringutils
+
+The regex-related APIs that were supported by this module have been moved to the new `regex` module. The rest of the APIs have replacements in the langlib packages.
+
+###### Socket
+
+The `socket` module was removed and got replaced by the `TCP` and `UDP` modules.
+
+#### Observability
+
+- A new extension model, which separates each extension into a separate module is introduced.
+    - Observability is not included into the final JAR by default. It can be added with the following configuration in the `Ballerina.toml`
+
+      ```toml
+      [build-options]
+      observabilityIncluded=true
+      ```
+
+    - The observability extension can be packaged by adding an import to the module in the code as shown in the example below.
+
+      ```ballerina
+      import ballerinax/prometheus as _;
+      ```
+
+- `Prometheus` and `Jaeger` extensions are introduced back.
+    - The `Prometheus` extension can be enabled by adding the following configuration in the `Config.toml` file.
+
+      ```toml
+      [ballerina.observe]
+      metricsEnabled=true
+      metricsReporter="prometheus"
+      
+      [ballerinax.prometheus]
+      host="127.0.0.1"  # Optional Configuration. Default value is localhost
+      port=9797         # Optional Configuration. Default value is 9797
+      ```
+
+    - The `Jaeger` extension can be enabled by adding the following config.
+
+      ```toml
+      [ballerina.observe]
+      tracingEnabled=true
+      tracingProvider="jaeger"
+      
+      [ballerinax.jaeger]
+      agentHostname="127.0.0.1"  # Optional Configuration. Default value is localhost
+      agentPort=6831             # Optional Configuration. Default value is 6831
+      ```
+
+    - By default, the `Jaeger` extension now publishes traces to the Jaeger Agent using the `jaeger.thrift` over the compact Thrift protocol.  
+
+#### Code to Cloud
+
+- The Kubernetes artifacts can be generated using the `--cloud=k8s` build option. The `import ballerina/cloud as _` is no longer required.
+- VS code plugin support for the `Kubernetes.toml`
+    - Code Completion based on the c2c specification
```

Comment:
```suggestion
    - Code Completion based on the c2c specification.
```

This period at the end seems to be inconsistent though. Shall we check and fix? @praneesha